src/data/climate.ts

```diff
@@ -15,12 +14,34 @@ export type HvacMode =
 
 export const CLIMATE_PRESET_NONE = "none";
 
-type ClimateAttributes = TranslationDict["state_attributes"]["climate"];
-export type HvacAction = keyof ClimateAttributes["hvac_action"];
-export type FanMode = keyof ClimateAttributes["fan_mode"];
+export type HvacAction =
+  | "off"
+  | "heating"
+  | "cooling"
+  | "drying"
+  | "idle"
+  | "fan";
+export type FanMode =
+  | "off"
+  | "on"
+  | "auto"
+  | "low"
+  | "medium"
+  | "high"
+  | "top"
+  | "middle"
+  | "focus"
+  | "diffuse";
 export type PresetMode =
-  | keyof ClimateAttributes["preset_mode"]
-  | typeof CLIMATE_PRESET_NONE;
+  | "none"
+  | "eco"
+  | "away"
+  | "boost"
+  | "comfort"
+  | "home"
+  | "sleep"
+  | "activity";
```

Comment:
@steverep I had to revert your changes as we now use the translation backend for attributes.